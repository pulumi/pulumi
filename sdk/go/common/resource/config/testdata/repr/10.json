{"my:servers":[{"host":"example","port":80}]}