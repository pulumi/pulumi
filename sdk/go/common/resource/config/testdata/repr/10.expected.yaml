decrypt:
  my:servers: '[{"host":"example","port":80}]'
paths:
  my:servers:
    value:
    - host: example
      port: 80
    string: '[{"host":"example","port":80}]'
    redacted: '[{"host":"example","port":80}]'
    object: HRAADltdaW50ZXJmYWNlIHt9/4MCAQL/hAABEAAAWP+EKAABF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAAAs/4IpAAIEaG9zdAZzdHJpbmcMCQAHZXhhbXBsZQRwb3J0BWludDY0BAMA/6A=
    secure: false
    isObject: true
  my:servers[0]:
    value:
      host: example
      port: 80
    string: '{"host":"example","port":80}'
    redacted: '{"host":"example","port":80}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAAAs/4IpAAIEcG9ydAVpbnQ2NAQDAP+gBGhvc3QGc3RyaW5nDAkAB2V4YW1wbGU=
    secure: false
    isObject: true
  my:servers[0].host:
    value: example
    string: example
    redacted: example
    object: FBAABnN0cmluZwwJAAdleGFtcGxl
    secure: false
    isObject: false
  my:servers[0].port:
    value: "80"
    string: "80"
    redacted: "80"
    object: DxAABnN0cmluZwwEAAI4MA==
    secure: false
    isObject: false
